user_configs_examples/sample_generate_sbatch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# specifies sbatch resources needed for the job
+
+#SBATCH --mem=16G
```

Comment:
@cpaniaguam @hannahhafner I think this should just be a basic bash file no?

Are the SBATCH parameters set here respected by the python script? Don't think so.

Reply:
Sidenote: Both the `.sh` files are not example `user_configs`, maybe `bash_script_examples` should be the folder?

Reply:
We could consider a `no-sbatch` mode (or one could possibly call it local/slurm), but ok for now. 


ok re example sbatch scripts, just as per our discussion on tuesday, we have to make sure that our command line tool actually produces exactly what we need so that no further intervention on manually created sbatch scripts is necessary..